### DISCUSSION TITLE
White line on taskbar

### DISCUSSION THREAD

**Discussion by IllusionerBow** [valinet/ExplorerPatcher#556] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89000977/145752384-9ed69125-bb69-4922-9ad7-1acc0f730e3d.png)
There is a white line on Windows 11's Taskbar style after installing ExplorerPatcher. This won't occur in Windows 10's Taskbar style.

**Comment by valinet** [comment, marked as answer]
I don’t experience that. Check in a clean VM. What Windows build?

**Reply by IllusionerBow** [reply]
The Windows build is 22000.348. The white line didn't occur in clean VM. Also, after I restore default setting(ExplorerPatcher), the white line disappear.

**Reply by IllusionerBow** [reply]
I find out the reason why there is a white line on Windows 11's Taskbar style. It is because i had turn on show separators between toolbars.

**Reply by valinet** [reply]
Okay, that’s great to hear, that you solved the issue.
